github/util_v4_branch_protection.go

```diff
@@ -293,6 +316,18 @@ func branchProtectionResourceDataActors(d *schema.ResourceData, meta interface{}
 			data.RestrictsPushes = true
 		}
 	}
+
+	if v, ok := d.GetOk(PROTECTION_FORCE_PUSHES_BYPASSERS); ok {
+		bypassForcePushActorIDs := make([]string, 0)
+		vL := v.(*schema.Set).List()
+		for _, v := range vL {
+			bypassForcePushActorIDs = append(bypassForcePushActorIDs, v.(string))
+		}
+		if len(bypassForcePushActorIDs) > 0 {
+			data.BypassForcePushActorIDs = bypassForcePushActorIDs
```

Comment:
Not sure if we need to set `data.AllowsForcePushes = true` here or not.

Reply:
I think we should. Otherwise that property won't be set at all and will default to false. It's only set in Create/Update operations and this is a Read operation, so it won't be set correctly later. Do you mind making that change?